modules/signals/src/signal-state.ts

```diff
@@ -2,7 +2,7 @@ import { signal } from '@angular/core';
 import { STATE_SIGNAL, StateSignal } from './state-signal';
 import { DeepSignal, toDeepSignal } from './deep-signal';
 
-type SignalState<State extends object> = DeepSignal<State> & StateSignal<State>;
+export type SignalState<State extends object> = DeepSignal<State> & StateSignal<State>;
```

Comment:
Could you give a use case where you require this type?
We try to expose as little as possible to lower the surface of future breaking changes to types.